JSTests/microbenchmarks/get-private-name.js

```diff
@@ -1,5 +1,5 @@
 //@ $skipModes << :lockdown if $buildType == "debug"
-//@ skip if $architecture == "arm" and !$cloop
+//@ skip if $memoryLimited and !$cloop
```

Comment:
this could probably just be $memoryLimited now

Reply:
Adressed in [3db23fb](https://github.com/WebKit/WebKit/pull/24415/commits/3db23fb6d5c65cd21f2fb6d34a0d146886608dcc)